src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -74,6 +74,16 @@ public AcquireTokenForClientParameterBuilder WithSendX5C(bool withSendX5C)
             return this;
         }
 
+        /// <summary>
+        /// Specifies that the certificate provided will be used for PoP tokens with MTLS (Mutual TLS) authentication.
```

Comment:
We need an aka.ms link. Maybe point to a (new?) section of https://learn.microsoft.com/en-us/entra/msal/dotnet/advanced/proof-of-possession-tokens

Reply:
@bgavrilMS @gladjohn - should I create a doc PR that we can sim-ship with this change? That way you will have documentation right as the feature goes out.